edb/testbase/server.py

```diff
@@ -289,6 +291,40 @@ async def connect(cls, *,
             cluster=cluster, database=database, user=user, password=password)
         return await edgedb.async_connect(**conargs)
 
+    def psql(self):
+        """Open interactive psql right in the test.
```

Comment:
Why `psql` and not `edgedb-cli`?

Reply:
Good idea, I've implemented that. The only problem is that it uses our legacy REPL because the new CLI doesn't recognize EDGEDB_PASSWORD env var https://github.com/edgedb/edgedb-cli/issues/57